docs/pages/access-controls/guides/dual-authz.mdx

```diff
@@ -170,7 +174,7 @@ output = "stderr" # Logger output. Could be "stdout", "stderr" or "/var/lib/tele
 severity = "INFO" # Logger severity. Could be "INFO", "ERROR", "DEBUG" or "WARN".
```

Comment:
I wonder what's up with the formatting here? 
![image](https://user-images.githubusercontent.com/559288/121928898-7bbd7600-ccf5-11eb-8e1f-62abe3bd7f06.png) 

You might want to double check on staging before deploying.

Reply:
Hmm, this is actually _unchanged_: https://github.com/gravitational/teleport/blob/branch/v6.2/docs/pages/access-controls/guides/dual-authz.mdx - I think sometimes we'll get false flags because of MDX components not rendering correctly (although this usually just hides a bunch of content and doesn't throw an error).

This renders correctly locally, doesn't throw a linting error, and can be seen fine live: https://goteleport.com/docs/access-controls/guides/dual-authz/

This is likely the result of the `[team/channel]` value not having `""`s.

I believe the new Slack article replaces these configuration files with new ones (for Slack). I'll likely swap these out as well.

Reply:
Updated to use the new TOML file.